DynamicGUI.py

```diff
@@ -154,7 +154,7 @@ def updateGrid(self):
                 self.visitedSet.add(curr_tile)
                 lidar_data = self.generate_sensor.generateLidar(
                     10, curr_tile.row, curr_tile.col)
-                if (self.gridEmpty.updateGridLidar(
+                if (self.gridEmpty.update_grid_lidar(
```

Comment:
@kennycai0 @kimmylin727 Change this to the right function call

Reply:
resolving